prep-css-display/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <style>
+  div {
+    border: 4px solid blue;
+  }
+  span {
+    border: 4px solid green;
+    height: 100px;
+    width: 100px;
+  }
+  button {
+    border: 4px solid red;
+    height: 40px;
+    width: 200px;
+  }
+  ul {
+    border: 4px solid purple;
+  }
+  li {
+    border: 4px solid magenta;
+    display: inline;
+  }
+  </style>
+<div>I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
+
+  <div>
+    I am a div but I have <span>span elements inside me</span>.
+  Span elements are <span>display inline by default</span>,
+  so they only take up as much as space as they need.
+  </div>
+
+  <div>
+    <button>Hello!</button>
+    <button>Buttons are</button>
+    <button>Inline Block</button>
+    <button>By Default</button>
+  </div>
+
+  <ul>
+    <li>
+      Home
+    </li>
+    <li>
+      Products
+    </li>
+    <li>
+      About Us
+    </li>
+    <li>
+      Contact Us
+    </li>
+  </ul>
+
+
+
+
+  <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Display Properties</title>
+  <style>
+
+  </style>
+  </head>
+
+<body>
+
+</body>
```

Comment:
Be sure to put all meta information, including internal stylesheets, in the `<head>` of the document and all visible content in the `<body>` of the document.